sunpy/net/hek/utils.py

```diff
@@ -0,0 +1,175 @@
+    for tkey in time_keys:
+        if tkey in table.colnames and not any(time == "" for time in table[tkey]):
+            table[tkey] = parse_time(table[tkey])
+            table[tkey].format = 'iso'
+    return table
+
+# NOTE: Needs unit test
+def parse_values_to_quantities(table):
+    with open(UNIT_FILE_PATH) as unit_file:
+        unit_properties = json.load(unit_file)
+    unit_attributes = unit_properties["attributes"]
+
+    with open(COORD_FILE_PATH) as coord_file:
+        coord_properties = json.load(coord_file)
+    coord_attributes = coord_properties["attributes"]
+    table = parse_columns_to_table(table, unit_attributes)
+    table = parse_columns_to_table(table, coord_attributes, is_coord_prop= True)
+    return table
+
+# NOTE: Needs unit test
+def parse_columns_to_table(table, attributes, is_coord_prop = False):
+    for attribute in attributes:
+        if attribute.get("is_unit_prop", False):
+            pass
+        elif attribute["name"] in table.colnames and ("unit_prop" in attribute or attribute.get("is_chaincode", False)):
+            table = parse_unit(table, attribute, is_coord_prop)
+            unit_attr = ""
+            if is_coord_prop:
+                unit_attr = "event_coordunit"
+            else:
+                unit_attr = attribute["unit_prop"]
+
+            new_column = []
+            for idx, value in enumerate(table[attribute["name"]]):
+                if value in ["", None]:
+                    new_column.append(value)
+                elif attribute.get("is_chaincode", False):
+                    new_column.append(parse_chaincode(value, idx, attribute, table[attribute["unit_prop"]][idx]))
+                else:
+                    new_column.append(value * get_unit(attribute["unit_prop"], table[unit_attr][idx]))
+            table[attribute["name"]] = new_column
+
+    for attribute in attributes:
+        if attribute.get("is_unit_prop", False) and attribute["name"] in table.colnames:
+            del table[attribute["name"]]
+    return table
+
+# NOTE: Needs unit test
+def parse_unit(table, attribute, is_coord_prop = False):
+    if attribute.get("is_chaincode", False):
+        return table
+    unit_attr = ""
+    if is_coord_prop:
+        unit_attr = "event_coordunit"
+    else:
+        unit_attr = attribute["unit_prop"]
+    for row in table:
+        if unit_attr in table.colnames and row[unit_attr] not in ["", None] and table[attribute["name"]].unit is not None:
+            table[attribute["name"]].unit = get_unit(attribute["unit_prop"], row[unit_attr])
+            break
+    return table
+
+# NOTE: Needs unit test
+def parse_chaincode(value, idx, attribute, unit_prop):
+    coord1_unit = u.deg
+    coord2_unit = u.deg
+    if attribute["frame"] == "helioprojective":
+        coord1_unit = u.arcsec
+        coord2_unit = u.arcsec
+    elif attribute["frame"] == "heliocentric":
+        coord1_unit = u.R_sun
+        coord2_unit = u.deg
+    elif attribute["frame"] == "icrs":
```

Comment:
Can you tell me which attributes give an icrs value?

Reply:
`Bound_ChainCode` and `Skel_ChainCode`

Reply:
Can you start a document or make a note of this? I want to make sure we capture any of this quirks but not via code comments.